util/deployment/deploy-production

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+echo "Starting production deployment..."
+
+# Ensure we are in the project root
+if [ ! -f "GEMINI.md" ] || [ ! -d "infra" ] || [ ! -d "frontend" ] || [ ! -d "backend" ]; then
+  echo "Please run this script from the root of the webstatus.dev repository."
+  exit 1
+fi
+
+echo "Building..."
+make build
+
+cd infra
+
+echo "Authenticating with gcloud..."
+gcloud auth login
+gcloud auth application-default login --project=web-compass-prod
+gcloud auth configure-docker europe-west1-docker.pkg.dev --quiet
+
+ENV_ID="prod"
+export TF_WORKSPACE=${ENV_ID}
```

Comment:
Given we already have variables for a lot of the things and common things between deploy-production and deploy-staging, I wonder if we should create a helper script?

Then we could call it like this:

```
$ deploy-env ID=prod project=web-compass-prod tfvars=.envs/prod.tfvars internalproject=webstatus-dev-internal-prod api=https://api.webstatus.dev/v1/features website=https://webstatus.dev/
```

Happy to make that a v2 change though.